package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "cedar",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "description": "Visualization framework for the ArcGIS Platform",
   "main": "index.js",
   "scripts": {
-    "prepublish": "node -e \"require('grunt').tasks(['prepublish']);\"",
-    "start": "node -e \"require('grunt').tasks(['default']);\"",
-    "test": "node -e \"require('grunt').tasks(['test']);\""
+    "start": "npm run build && npm run server",
+    "build": "npm run compile && npm run bundle && npm run combine",
+    "compile": "babel src -d _tmp --blacklist es6.modules -e 0",
+    "bundle": "rollup _tmp/index.js -o dist/builds/cedar.js",
```

Comment:
I think this needs to be:

`"bundle": "rollup _tmp/cedar.js -o dist/builds/cedar.js",`